src/services/DappStakingV3IndexerBase.ts

```diff
@@ -12,6 +12,6 @@ export class DappStakingV3IndexerBase extends ServiceBase {
 
     protected getApiUrl(network: NetworkType): string {
         // For local development: `http://localhost:4350/graphql`;
-        return `https://astar-network.squids.live/dapps-staking-indexer-${network}/graphql`;
+        return `https://astar-network.squids.live/dapps-staking-indexer-${network}/v/v14/graphql`;
```

Comment:
Can you change the v14 to prod and revert this change?

Reply:
No, because of 
<img width="755" alt="image" src="https://github.com/user-attachments/assets/74027e06-b62b-4ca9-a9a7-f260675ce4c7">

Reply:
Ah okay